src/completion-provider.js

```diff
@@ -110,6 +116,28 @@ class CompletionProvider {
       (suggestions) => this.filterSuggestions(prefix, suggestions)
     );
   }
+
+  lookupWebpack(prefix) {
+    const projectPath = atom.project.getPaths()[0];
+    if (!projectPath) {
+      return Promise.resolve([]);
+    }
+
+    const webpackConfigFilename = atom.config.get('autocomplete-modules.webpackConfigFilename');
+    const webpackConfigPath = path.join(projectPath, webpackConfigFilename);
+    const vendors = atom.config.get('autocomplete-modules.vendors');
+
+    const webpackConfig = require(webpackConfigPath);
+    let moduleSearchPaths = get(webpackConfig, 'resolve.modulesDirectories', []);
```

Comment:
Hm... I think you should just wrap it into `try-catch` statement. For example, webpack config could use some new syntax which unsupported by current atom JS runtime.